### PR TITLE
chore: add avd_reference to kube-hunter report

### PR DESCRIPTION
**Pull request by krol3**
Related the issue [395](https://github.com/aquasecurity/starboard/issues/395)
Is it necessary to update the lens and octant plugin too?

### CODE REVIEW

File: pkg/apis/aquasecurity/v1alpha1/kube_hunter_types.go [hunk 1]
```diff
@@ -63,8 +65,91 @@ var (
...
+																{Raw: []byte(strconv.Quote(string(KubeHunterSeverityHigh)))},
+																{Raw: []byte(strconv.Quote(string(KubeHunterSeverityMedium)))},
+																{Raw: []byte(strconv.Quote(string(KubeHunterSeverityLow)))},
+																{Raw: []byte(strconv.Quote(string(KubeHunterSeverityUnknown)))},
+															},
+														},
+														"vulnerability": {Type: "string"},
+														"description":   {Type: "string"},
+														"evidence":      {Type: "string"},
+														"avdReference":  {Type: "string"},
```
Comment (danielpacak) [inline pkg/apis/aquasecurity/v1alpha1/kube_hunter_types.go:145]: KubeHunter is using `avd_reference` not `avdReference`. If you provide Open API spec only defined properties are persisted in CRD/etcd.

```suggestion
														"avd_reference":  {Type: "string"},
```

File: deploy/crd/kubehunterreports.crd.yaml [hunk 2]
```diff
@@ -31,8 +31,94 @@ spec:
...
+                        enum:
+                          - high
+                          - medium
+                          - low
+                          - unknown
+                      description:
+                        type: string
+                      evidence:
+                        type: string
+                      avdReference:
```
Comment (danielpacak) [inline deploy/crd/kubehunterreports.crd.yaml:120]:
```suggestion
                      avd_reference:
```

File: deploy/crd/kubehunterreports.crd.yaml [hunk 3]
```diff
@@ -31,8 +31,94 @@ spec:
...
+                    type: object
+                    required:
+                      - location
+                      - vid
+                      - category
+                      - severity
+                      - vulnerability
+                      - description
+                      - evidence
+                      - avdReference
```
Comment (danielpacak) [inline deploy/crd/kubehunterreports.crd.yaml:99]:
```suggestion
                      - avd_reference
```

**Review by danielpacak — CHANGES_REQUESTED** [review]
Good job @krol3 I think we just have to rename `avdRefernec` to `avd_reference` so the property is actually persisted in etcd.  Make sure it's the case by running `./bin/starboard scan kubehunterreports` and then inspect the actual report.

**Comment by krol3** [comment]
Hi! @danielpacak  I tested this new version, after the update.

`k get kubehunterreports.aquasecurity.github.io -o yaml | more`

```
    vulnerabilities:
    - avd_reference: https://avd.aquasec.com/kube-hunter/none/
      category: Access Risk
      description: |-
        CAP_NET_RAW is enabled by default for pods.
            If an attacker manages to compromise a pod,
            they could potentially take advantage of this capability to perform network
            attacks on other pods running on the same node
```

**Review by danielpacak — APPROVED** [review]

**Pull request merged**